pkg/registry/thirdpartyresourcedata/codec.go

```diff
@@ -32,6 +32,27 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
+type thirdPartyObjectConverter struct {
+	converter runtime.ObjectConvertor
+}
+
+func (t *thirdPartyObjectConverter) ConvertToVersion(in runtime.Object, outVersion string) (out runtime.Object, err error) {
+	switch in.(type) {
+	case *extensions.ThirdPartyResourceData:
+		return in, nil
```

Comment:
Can I get a comment about why this is correct?  Seems like you should check for this conflicting with `outVersion` and you not having any way to convert.